docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -1275,6 +1275,21 @@ The Tooltip's prop `componentsProps` was deprecated in favor of `slotProps`:
  />
 ```
 
+## Typography
+
+There is no codemod to manage this change.
```

Comment:
Same as in the `ListItemSecondaryAction` section in this file, we can omit that there's no codemod.

```suggestion
```